src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+const listItems = [...tree.querySelectorAll('li')];
+
+listItems.forEach((elem) => {
+  const firstText = elem.firstChild;
```

Comment:
The code assumes that `elem.firstChild` is always a text node. Consider adding a check to verify that `firstText` is indeed a text node before accessing `textContent`.